<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Research | Srivastava Lab</title>
    <meta name="author" content="Avi  Srivastava">
    <meta name="description" content="The webpage for Srivastava Lab at the Wistar institute.
">
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website">


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Bootstrap Table -->
    <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light">

    

    <!-- Styles -->
    
    <link rel="shortcut icon" href="/assets/img/wistar.ico">
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://avisrilab.org/research/">

    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark">

    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="/">Srivastava Lab</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">Home</a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item active">
                <a class="nav-link" href="/research/">Research<span class="sr-only">(current)</span></a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/publications/">Publications</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/people/">People</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/contact/">Join/Contact</a>
              </li>
<!-- Github Icon -->
              <li class="nav-item active">
                <a class="nav-link" href="https://github.com/avisrilab" rel="external nofollow noopener" target="_blank">
                  <i class="fab fa-github"></i></a>
              </li>

              <!-- Twitter Icon -->
              <li class="nav-item active">
                <a class="nav-link" href="https://twitter.com/k3yavi" rel="external nofollow noopener" target="_blank">
                  <i class="fab fa-twitter"></i></a>
              </li>

              <!-- Toogle theme mode -->
              <li class="toggle-container">
                <button id="light-toggle" title="Change theme">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </button>
              </li>
            </ul>
          </div>
        </div>
      </nav>

      <!-- Scrolling Progress Bar -->
      <progress id="progress" value="0">
        <div class="progress-container">
          <span class="progress-bar"></span>
        </div>
      </progress>
    </header>


    <!-- Content -->
    <div class="container mt-5">
      
        <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">Research</h1>
            <p class="post-description"></p>
          </header>

          <article>
            <p>Welcome to Srivastava Lab, where our unwavering <u>passion</u> drives us to tackle classical biological problems in modern ways. With a particular focus on employing novel biotechnologies and customized algorithmic solutions, our team of interdisciplinary scientists is committed to delving deep into complex challenges, breaking them down into manageable subproblems, and fostering a collaborative research environment.</p>

<p>Our <u>profession</u> revolves around identifying key biological problems with the potential to make a significant impact. At the forefront of our research lies a comprehensive exploration of how the epigenome regulates transcription, ultimately determining cell fate. Leveraging the power of single-cell biology, we propose computational and technological solutions that pave the way for a deeper understanding of this intricate process.</p>

<p>Yet, our purpose goes beyond theoretical discoveries. Our <u>vocation</u> lies in the application of these proposed solutions to real-world biological problems, especially those pertaining to the chromatin dynamics of blood cell development and its aberration in malignant conditions.</p>

<p>At the core of our endeavors is a steadfast <u>mission</u> to foster a healthier world. With the knowledge of disease mechanisms, we are driven to unlock the secrets behind aberrant conditions. By challenging conventional approaches, we pioneer novel targets and algorithms aimed at correcting and transforming the landscape of disease.</p>

<p>We firmly believe that the convergence of passion, profession, vocation, and mission is the catalyst for meaningful change and we continue to push the boundaries of science under the following broad but unifying themes of research:</p>

<font size="+2"> 1. Fast and efficient methods for bulk RNA-seq quantification</font>
<article>
<div class="profile float-right">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/salmon_logo-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/salmon_logo-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/salmon_logo-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/salmon_logo.png" class="img-fluid z-depth-0 rounded" width="auto" height="auto" alt="/assets/img/publication_preview/salmon_logo.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>
<div class="clearfix"> The accuracy of transcript quantification using RNA-seq data depends on many factors, such as the choice of alignment or mapping method and the quantification model. The alignment of sequencing reads to a transcriptome is a common and important step in many RNA-seq analysis tasks. While the choice of quantification model is important, considerably less attention has been given to the effect of various read alignment approaches on quantification accuracy. Thus, we investigated the influence of mapping and alignment of RNA-seq reads on the accuracy of transcript quantification and designed multiple novel alignment methodologies to overcome the shortcomings of lightweight approaches without incurring the computational cost of traditional end-to-end alignment.</div></article>
<font size="-1"> <p>* <a href="https://www.nature.com/articles/s41592-022-01408-3" rel="external nofollow noopener" target="_blank">He, Dongze, et al. "Alevin-fry unlocks rapid, accurate and memory-frugal quantification of single-cell RNA-seq data." Nature Methods 19.3 (2022): 316-322.</a><br>
* <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02151-8" rel="external nofollow noopener" target="_blank">Srivastava A., Malik L., Sarkar H., Zakeri M., Almodaresi F., Soneson C., Love MI., Kingsford C., &amp; Patro R. (2020) "Alignment and mapping methodology influence transcript abundance estimation." Genome Biology. 2020 Dec;21(1):1-29.</a><br>
* <a href="https://arxiv.org/abs/1604.03250" rel="external nofollow noopener" target="_blank">Srivastava A., Sarkar H., Malik L. &amp; Patro R. (2016) "Accurate, fast and lightweight clustering of de novo transcriptomes using fragment equivalence classes." RECOMB-seq Conference. 2016 Apr 12.</a><br>
* <a href="https://academic.oup.com/bioinformatics/article/32/12/i192/2288985" rel="external nofollow noopener" target="_blank">Srivastava A., Sarkar H., Gupta N. &amp; Patro R. (2016) "RapMap: a rapid, sensitive, and accurate tool for mapping RNA-seq reads to transcriptomes." Bioinformatics. 2016 Jun 15;32(12):i192-200.</a></p></font>

<font size="+2"> 2.	Uncertainty aware Bayesian methods improves scRNA-seq quantification</font>
<article>
<div class="profile float-right">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/alevin-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/alevin-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/alevin-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/alevin.jpeg" class="img-fluid z-depth-0 rounded" width="auto" height="auto" alt="/assets/img/publication_preview/alevin.jpeg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>
<div class="clearfix"> There has been a steady increase in the throughput of single-cell (sc)RNA-seq experiments, facilitating experimental assay of millions of cells. Droplet based scRNA-seq experiments have a large set of gene-ambiguous reads and can commonly account for a quarter of the sequenced data, which stays largely unused by quantification methods. We designed alevin, a fast end-to-end pipeline to process scRNA-seq data, performing cell barcode detection, read mapping, UMI deduplication, gene count estimation, and cell barcode whitelisting. Alevin’s approach to UMI deduplication provides an uncertainty-aware Bayesian model to account for reads that multimap between genes. This addresses the inherent bias in existing tools which discard gene-ambiguous reads and improves the accuracy of gene abundance estimates.</div></article>
<font size="-1"> <p>* <a href="https://academic.oup.com/bioinformatics/article/38/10/2773/6564225" rel="external nofollow noopener" target="_blank">Mu, Wancen, et al. "Airpart: interpretable statistical models for analyzing allelic imbalance in single-cell datasets." Bioinformatics 38.10 (2022): 2773-2780.</a><br>
* <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008585" rel="external nofollow noopener" target="_blank">Soneson C., Srivastava A., Patro R. &amp; Stadler MB. (2021) "Preprocessing choices affect RNA velocity results for droplet scRNA-seq data." PLOS Computational Biology. 2021 Jan 11;17(1):e1008585.</a><br>
* <a href="https://link.springer.com/article/10.1186/s13059-019-1670-y" rel="external nofollow noopener" target="_blank">Srivastava A., Malik L., Smith T., Sudbery I. &amp; Patro R. (2019) "Alevin efficiently estimates accurate gene abundances from dscRNA-seq data." Genome biology. 2019 Dec;20(1):1-6.</a><br>
* <a href="https://academic.oup.com/nar/article/47/18/e105/5542870" rel="external nofollow noopener" target="_blank">Zhu A., Srivastava A., Ibrahim JG., Patro R. &amp; Love MI. (2019) "Nonparametric expression analysis using inferential replicate counts." Nucleic Acids Research. 2019 Oct 10;47(18):e105.</a></p></font>

<font size="+2"> 3.	Computational Methods for single-cell analyses</font>
<article>
<div class="profile float-right">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/scnt-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/scnt-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/scnt-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/scnt.png" class="img-fluid z-depth-0 rounded" width="auto" height="auto" alt="/assets/img/publication_preview/scnt.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>
<div class="clearfix"> scRNA-seq data is being generated at an unprecedented pace, and the accurate estimation of gene-level abundances for each cell is a crucial first step in most scRNA-seq analyses. When pre-processing the raw scRNA-seq data to generate a count matrix, care must be taken to account for the potentially large number of multi-mapping locations per read. The sparsity of scRNA-seq data, and the strong 3’ sampling bias, make it even more challenging to disambiguate cases where there is no uniquely mapped read to any of the candidate target genes. We introduced a Bayesian framework for information sharing across cells within a sample or across multiple modalities of data to improve gene quantification estimates for scRNA-seq data.</div></article>
<font size="-1"> <p>* <a href="https://www.nature.com/articles/s41587-023-01767-y" rel="external nofollow noopener" target="_blank">Hao, Yuhan, et al. "Dictionary learning for integrative, multimodal and scalable single-cell analysis." Nature Biotechnology (2023): 1-12.</a><br>
* <a href="https://www.nature.com/articles/s41587-022-01250-0" rel="external nofollow noopener" target="_blank">Zhang, B.*, Srivastava A.*, Mimitou E., Stuart T., Raimondi I., Hao Y., Smibert P. &amp; Satija R. (2021) "Characterizing cellular heterogeneity in chromatin state with scCUT\&amp;Tag-pro." Nature Biotechnology 40.8 (2022): 1220-1230. </a><br>
* <a href="https://www.nature.com/articles/s41592-021-01282-5" rel="external nofollow noopener" target="_blank">Stuart T., Srivastava A., Lareau C. &amp; Satija R. (2021) "Single-cell chromatin state analysis with Signac." Nature Methods (2021): 1-9.</a><br>
* <a href="https://academic.oup.com/bioinformatics/article/36/Supplement_1/i292/5870502" rel="external nofollow noopener" target="_blank">Srivastava A., Malik L., Sarkar H. &amp; Patro R. (2020) "A Bayesian framework for inter-cellular information sharing improves dscRNA-seq quantification." Bioinformatics. 2020 Jul 1;36(Supplement\_1):i292-9.</a></p></font>

<font size="+2"> 4. Integrated analyses of the epigenome to understand the molecular basis of hematopoietic malignancies</font>
<article>
<div class="profile float-right">
<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/publication_preview/k99-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/publication_preview/k99-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/publication_preview/k99-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/publication_preview/k99.jpeg" class="img-fluid z-depth-0 rounded" width="auto" height="auto" alt="/assets/img/publication_preview/k99.jpeg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>
</div>
<div class="clearfix"> An impaired hematopoietic differentiation process underlies bone marrow malignancies like leukemia, but we still lack the mechanistic understanding of the sequence of regulatory events that misleads the differentiation process. Since epigenomic regulatory patterns are major features of leukemic development, understanding the chromatin dynamics of a failed (malignant) hematopoietic differentiation process can help define the molecular basis of leukemia. A prerequisite to such an understanding is a framework that allows investigation of the progressive changes in the activity of the regulatory elements (RE) during hematopoietic differentiation. Single-cell CUT&amp;Tag (scCUT&amp;Tag) technology is well-suited for such studies as RE activity through histone modification profiles can be investigated in a lineage-specific manner. Using scCUT&amp;Tag we will investigate the RE and progressive changes in their activity during hematopoiesis. First, we will define a multimodal reference mapping framework for mouse hematopoiesis. This framework will allow us to integrate multiple histone modification profiles onto one reference and compare the chromatin states of the RE between a wild type (WT) and mouse model with loss of function in histone methyl transferase (HMT). Second, since HMTs regulate transcription through the interaction network of RE. We will define a chromatin state aware map that dynamically links REs across developmental trajectories. We will use this framework to investigate the changes in the interaction of REs due to HMT loss. Third, since the transcriptional state of a cell emerges from the underlying gene regulatory network (GRN), We will integrate single-cell gene expression data with histone modification profiles and extend it to define a chromatin state aware model of GRN. We will compare the WT and HMT loss experiments and define the differential GRN.</div></article>
<font size="-1"> <p>* <a href="https://reporter.nih.gov/project-details/10538621" rel="external nofollow noopener" target="_blank">Srivastava A. (2020) "Integrated analyses of the epigenome to understand the molecular basis of hematopoietic malignancies.", Project Number: K99CA267677.</a></p></font>


          </article>

        </div>

      
    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        © Copyright 2023 Avi  Srivastava. Powered by <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme.
Last updated: September 13, 2023.
      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js"></script>

  <!-- Bootstrap Table -->
  <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script>

  <!-- Load Common JS -->
  <script src="/assets/js/no_defer.js"></script>
  <script defer src="/assets/js/common.js"></script>
  <script defer src="/assets/js/copy_code.js" type="text/javascript"></script>

    
  <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
  <script async src="https://badge.dimensions.ai/badge.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
    

<!-- Scrolling Progress Bar -->
<script type="text/javascript">
  /*
   * This JavaScript code has been adapted from the article 
   * https://css-tricks.com/reading-position-indicator/ authored by Pankaj Parashar, 
   * published on the website https://css-tricks.com on the 7th of May, 2014.
   * Couple of changes were made to the original code to make it compatible 
   * with the `al-foio` theme.
   */
  const progressBar = $("#progress");
  /*
   * We set up the bar after all elements are done loading.
   * In some cases, if the images in the page are larger than the intended
   * size they'll have on the page, they'll be resized via CSS to accomodate
   * the desired size. This mistake, however, breaks the computations as the
   * scroll size is computed as soon as the elements finish loading.
   * To account for this, a minimal delay was introduced before computing the
   * values.
   */
  window.onload = function () {
    setTimeout(progressBarSetup, 50);
  };
  /*
   * We set up the bar according to the browser.
   * If the browser supports the progress element we use that.
   * Otherwise, we resize the bar thru CSS styling
   */
  function progressBarSetup() {
    if ("max" in document.createElement("progress")) {
      initializeProgressElement();
      $(document).on("scroll", function() {
        progressBar.attr({ value: getCurrentScrollPosition() });
      });
      $(window).on("resize", initializeProgressElement);
    } else {
      resizeProgressBar();
      $(document).on("scroll", resizeProgressBar);
      $(window).on("resize", resizeProgressBar);
    }
  }
  /*
   * The vertical scroll position is the same as the number of pixels that
   * are hidden from view above the scrollable area. Thus, a value > 0 is
   * how much the user has scrolled from the top
   */
  function getCurrentScrollPosition() {
    return $(window).scrollTop();
  }

  function initializeProgressElement() {
    let navbarHeight = $("#navbar").outerHeight(true);
    $("body").css({ "padding-top": navbarHeight });
    $("progress-container").css({ "padding-top": navbarHeight });
    progressBar.css({ top: navbarHeight });
    progressBar.attr({
      max: getDistanceToScroll(),
      value: getCurrentScrollPosition(),
    });
  }
  /*
   * The offset between the html document height and the browser viewport
   * height will be greater than zero if vertical scroll is possible.
   * This is the distance the user can scroll
   */
  function getDistanceToScroll() {
    return $(document).height() - $(window).height();
  }

  function resizeProgressBar() {
    progressBar.css({ width: getWidthPercentage() + "%" });
  }
  // The scroll ratio equals the percentage to resize the bar
  function getWidthPercentage() {
    return (getCurrentScrollPosition() / getDistanceToScroll()) * 100;
  }
</script>

  </body>
</html>
